src/components/Links/Links.scss

```diff
@@ -0,0 +1,10 @@
+section {
+    nav {
+        margin-top: 20px;
+    }
+
+    .text-link-icon {
+        color: #23419f;
+        margin-right: 7px;
```

Comment:
Instead of including `margin-right` here, you could use Bootstrap's class `.mr-2` on the `Icon` component.